<launch>
    <arg name="field_name" default="test01" />
    <arg name="experiment_name" default="recorrida1" />

    <node name="record_images_node" pkg="fields_ignition" type="record.py" output="screen">
        <param name="field_name" value="$(arg field_name)" />
        <param name="experiment_name" value="$(arg experiment_name)" />
    </node>

</launch>